### ISSUE TITLE
Conflicts with https://github.com/actions/labeler ? 

### ISSUE THREAD

**Issue by thollander** [Decathlon/pull-request-labeler-action#38]
Is this project deprecated now that there is an [official labeler](https://github.com/actions/labeler) from github actions ?

**Comment by rdelgatte** [comment]
I guess it should be to avoid any duplications (even though there are already): 
![image](https://user-images.githubusercontent.com/11570518/91011530-c34b0800-e5e4-11ea-88ff-cf45ba846835.png)

@lusoalex @mmornati should I deprecate this repository in favor of the one @thollander refers to?

**Comment by mmornati** [comment]
Yes, frustrating as we did it before :) I think we can check if it is the same and in case we can maybe deprecate it in favor of the official one. what about you @lusoalex ?

**Comment by lusoalex** [comment]
Yes, I am sad but that's the open-source life...
There's no reason to duplicate actions if they deliver the same service.
We should archive our repository and add a comment to point on the official / most recommended one.
